src/main/java/nl/tudelft/ti2206/group9/shop/ShopItemUnlocker.java

```diff
@@ -0,0 +1,83 @@
+package nl.tudelft.ti2206.group9.shop;
+
+import java.util.HashMap;
+
+/**
+ * This class is responsible for maintaining a HashMap containing
+ * all data about which shop items are unlocked and which aren't.
+ * The HashMap is kept separated from the Shop Items so that
+ * the HashMap can be altered during the tests which involve unlocking
+ * shop items. (If also the textures are loaded during the tests,
+ * (graphics) errors will occur.)
+ * @author Mitchell
+ */
+public final class ShopItemUnlocker {
+
+    /** HashMap used to store which shop items are unlocked. */
+    private static HashMap<String, Boolean> unlockedShopItemsMap =
+            new HashMap<>();
```

Comment:
Thank you noticing. I've changed the HashMap to a Map. Besides, instead of basing this Map on a HashMap, I'm using a ConcurrentHashMap now, as we are using Java 5+.

<review,interface